.travis.yml

```diff
@@ -118,6 +118,10 @@ script:
         cmake $MY_CMAKE_OPTIONS . &&
         make &&
         make install ); fi
+  # Check that all installed files are in a component (no "unspecified
+  # component" archive created)
+  - (cd build && cpack -G TGZ -D CPACK_ARCHIVE_COMPONENT_INSTALL=ON &&
+     [ ! -x '*-Unspecified.tar.gz' ])
```

Comment:
I think you want to use `-a` instead of `-x`
```
$ help test
    -a FILE        True if file exists.
   ...
    -x FILE        True if the file is executable by you.
   ...
```